pbs_light.go

```diff
@@ -106,6 +108,7 @@ var (
 	accountMetricsRWMutex sync.RWMutex
 
 	hostCookieSettings pbs.HostCookieSettings
+	analytics          *a.PBSAnalyticsModule
```

Comment:
Not fond of global variables, also should be renamed so you don't have to rename the analytics module the undescriptive "a". Given the state of the legacy code, it may be more work than its worth to pass the analytics module object all the way through. Glad to see the analytics module attached to the OpenRTB exchange object, will make removing the global easier if and when legacy goes away.

I would recommend calling the global something like pbsAnalytics, and leaving the package name unaliased, with the understanding that the global variable may some day go away. This way we have a final state of the analytics package not being aliased.